api4/webhook_test.go

```diff
@@ -719,7 +719,7 @@ func TestGetOutgoingWebhook(t *testing.T) {
 	CheckForbiddenStatus(t, resp)
 
 	th.TestForSystemAdminAndLocal(t, func(t *testing.T, client *model.Client4) {
-		nonExistentHook := &model.OutgoingWebhook{ChannelId: th.BasicChannel.Id}
+		nonExistentHook := &model.OutgoingWebhook{}
```

Comment:
Why was this changed?

Reply:
It was one of the linting changes I mentioned in the PR description. If I were to do it again I'd split them into a separate PR for ease of reviewing, but anyhow.